packages/app/resources/lang/de/pages/auth/register.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+
+    'title' => 'Registrierung',
```

Comment:
Wouldn't you want this to match the heading?

Reply:
They don't match in english either, but the title is now following the same convention as on Login.

Reply:
Yeah it's a little different in English.
Dutch translations I've added today are similar to how you've approached it for German now.
Looks good to me.